src/hooks/useResource.js

```diff
@@ -27,52 +30,59 @@ const useResource = (errorKey, errorRef) => {
   const resourceUriMap = useSelector((state) => selectResourceUriMap(state))
 
   const [navigateEditor, setNavigateEditor] = useState(false)
+  const [status, setStatus] = useState("ready")
 
-  useLayoutEffect(() => {
+  useEffect(() => {
     // Forces a wait until the root resource has been set in state
     if (navigateEditor && resourceKey && _.isEmpty(errors)) {
       history.push("/editor")
-    } else if (!_.isEmpty(errors) && errorRef) {
```

Comment:
I got rid of the scrolling to errors. If proves to be a problem, I'd probably add the scrolling directly to Alert, rather than here.

Reply:
wondering if this comment should be captured somewhere more useful, like in an issue "if xxx is a problem, then"